.github/workflows/testoperator_dispatch.yml

```
@@ -44,28 +44,38 @@ jobs:
   dispatch-tests:
     name: Dispatch tests
     runs-on: spiceai-runners
+    concurrency:
+      group: testoperator-dispatch
+      cancel-in-progress: false
```

I don't think we should enable concurrency like this. It's completely valid for more than one to be running at a time, scheduling under different paths. I think a more relevant concurrency group would be:

```suggestion
    concurrency:
      group: testoperator-dispatch-${{ github.events.inputs.test_files_path || 'scheduled' }
      cancel-in-progress: false
```